www/src/data/sidebars/doc-links.yaml

```diff
@@ -178,14 +178,17 @@
     - title: Debugging
       link: /docs/debugging/
       items:
-        - title: Debugging HTML builds
-          link: /docs/debugging-html-builds/
-        - title: Debugging Replace Renderer API
-          link: /docs/debugging-replace-renderer-api/
-        - title: Debugging the build process
+        - title: Debugging in development
           link: /docs/debugging-the-build-process/
-        - title: Trace Gatsby builds
-          link: /docs/performance-tracing/
+        - title: Debugging (builds) in production
```

Comment:
This is tough to name correctly and concisely - it targets errors happening during server side rendering (so when actually saving .html files) specifically

Reply:
So it will happen in production but not development, right? If there are other classes of errors that only happen in production, we can add them to this document.

How can we best emphasize "these are errors that only happen in prod, not development"? Maybe "Debugging production builds"?

Reply:
The first doc is a general guide on 'how to debug the build process', the second doc is specific advice on fixing 'common' types of build errors. How about these as the titles?

- How to debug your Gatsby builds
- Fixing common `gatsby build` errors

I don't love using the word 'common' to reference the frequency of errors :) but couldn't come up with anything better.

Reply:
What's the status here? :)